apps/condo/domains/acquiring/schema/AcquiringIntegration.js

```diff
@@ -73,19 +73,11 @@ const AcquiringIntegration = new GQLListSchema('AcquiringIntegration', {
             isRequired: true,
         },
 
-        supportedBillingIntegrations: {
-            schemaDoc: 'List of supported billing integrations. If one of them is here, it means that this acquiring can accept receipts from it',
-            type: Relationship,
-            ref: 'BillingIntegration',
+        supportedBillingIntegrationsGroup: {
+            schemaDoc: 'Supported billing integrations group. Useful when you need to restrict this acquiring to accept payment only from certain billing',
```

Comment:
Logic on how it works would be awesome...Took some time to figure it out from code...And will take a lot more from pure schemaDoc...

JSDoc near common, default values inside constants are also appreciated

Reply:
Yep, jsdoc will be added